content/en/docs/topics/architecture.md

```diff
@@ -5,14 +5,11 @@ aliases: ["/docs/architecture/"]
 weight: 8
 ---
 
-# The Kubernetes Helm Architecture
```

Comment:
Should this heading be restored?

Reply:
The header is being displayed... is there a problem?

https://deploy-preview-732--helm-merge.netlify.app/docs/topics/architecture/

Reply:
I believe that what's being displayed now is the "Helm Architecture" title from Hugo's frontmatter for this page. We no longer have the H1 title. Given that it's a bit redundant as well as using the old "Kubernetes Helm" language, it's probably fine to get rid of it. I want to make sure @bacongobbler definitely intends to remove it instead of, perhaps, just removing the word "Kubernetes".

Old:
![image](https://user-images.githubusercontent.com/2104453/86257176-d7433200-bb7e-11ea-9afb-56f00b40cac3.png)

New:

![image](https://user-images.githubusercontent.com/2104453/86257201-e0cc9a00-bb7e-11ea-9b97-07107eec494b.png)